### PR TITLE
fix(STONEINTG-527): fix the name of the workspace

### PR DESCRIPTION
**Pull request by jinqi7**

### CODE REVIEW

**Review by MartinBasti — APPROVED** [review]

**Review by dirgim — APPROVED** [review]
LGTM

**Review by jsztuka — APPROVED** [review]
lgtm

File: task/fbc-validation/0.1/fbc-validation.yaml [hunk 1]
```diff
@@ -84,8 +84,8 @@ spec:
 
         ### Try to extract binaries with configs > check binaries functionality > check opm validate ###
         if [ ! -s ../inspect-image/image_inspect.json ]; then
-          echo "File $(workspaces.source.path)/hacbs/inspect-image/image_inspect.json did not generate correctly. Check inspect-image task log."
```
Comment (hongweiliu17) [inline task/fbc-validation/0.1/fbc-validation.yaml:87]: I am confused here. Doesn't the `source` represent the workspace name of task `inspect-image` in https://github.com/redhat-appstudio/build-definitions/blob/main/task/inspect-image/0.1/inspect-image.yaml#L33, is it a right path if we change to `$(workspaces.workspace.path)` here? @jinqi7 @dirgim @MartinBasti

Comment (jsztuka) [reply task/fbc-validation/0.1/fbc-validation.yaml:87]: I think that is the case, yes.

**Pull request merged**